src/charm.py

```diff
@@ -154,11 +156,26 @@ def __init__(self, *args, **kargs) -> None:
             self.unit.status = ops.BlockedStatus(exc.msg)
             return
 
+        self._event_timer = EventTimer(self.unit.name)
+        if not self._event_timer.is_active(RECONCILE_RUNNERS_EVENT):
```

Comment:
What happens if this command for checking if is active fails?

Reply:
Thx, nice catch.